students/jstonham/README.md

```diff
@@ -2648,11 +2691,14 @@ Web servers are not required to follow these methods, and can use methods (e.g.
 
 When sent from the client, methods such as `POST`, `PUT` and `PATCH` always consist of a header and a body (as the client needs to send the web server the data it wants to add or update).
 
-The `GET` method should not have a body when sent from the client as additional information can be included in the Uniform Resource Locator (URL).
+[Request for Comments](https://www.ietf.org/standards/rfcs/) specifies that the `GET` method should not have a body when sent from the client.
```

Comment:
I was betting on this being true during our session. However [the truth seems to be a bit more convoluted](https://stackoverflow.com/a/983458) Maybe reword this sentence to link to this stackoverflow answer instead

```suggestion
The HTTP [Request for Comments](https://www.ietf.org/standards/rfcs/) [vaguely specifies](https://stackoverflow.com/a/983458) that the a body in a `GET` request should not alter the behaviour of the server in any way .
```